src/mongocxx/options/insert.hpp

```diff
@@ -42,6 +43,20 @@ class MONGOCXX_API insert {
     void write_concern(class write_concern wc);
 
     ///
+    /// @note: This applies only to insert_many and is ignored for insert_one.
+    ///
+    /// If true, when an insert fails, return without performing the remaining
+    /// writes. If false, when a write fails, continue with the remaining
+    /// writes, if any. Defaults to true.
```

Comment:
I'd add that in addition to the failure semantics, inserts could be performed in any order if "ordered" is false.